src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -164,9 +166,14 @@ public DynamicACLFilter(final UserProjectCache cache, final Settings settings, f
         this.esClient = TransportClient.builder().settings(clientSettings).addPlugin(SearchGuardSSLPlugin.class)
                 .addPlugin(SearchGuardPlugin.class) // needed for config update
                                                     // action only
-                .build().addTransportAddress(new InetSocketTransportAddress(new InetSocketAddress("localhost", 9300)));
+                .build();
+        try {
+            this.esClient.addTransportAddress(new InetSocketTransportAddress(new InetSocketAddress("localhost", 9300)));
```

Comment:
Do we have to be concerned here if port 9300 is not used by ES?  How would this plugin know?